### ISSUE TITLE
Suggestion: Deep-dive content

### ISSUE THREAD

**Issue by arafattehsin** [sjwaight/AzureLinks#1]
Great resources on Azure to start with. However, I have got a suggestion.

How about having a folder structure and covering each section with their specific links / resources. For example;

- Azure Bot Service
- Azure Machine Learning
- Azure Cognitive Services
- So on..

**Comment by sjwaight** [comment]
How about if I created another page that contained per-service categories with links to useful quickstarts and training? I'm not trying to recreate docs.microsoft.com or Microsoft Learn here, so need to measure up the usefulness of such content.

**Comment by arafattehsin** [comment]
I agree with you on this. Over here, you can share the external links as well such as MVP blogs, tutorials or YouTube videos on the same topic which are not there in the docs.
